tests/php/validation/test-class-amp-validation-error-taxonomy.php

```diff
@@ -1429,27 +1431,41 @@ static function ( $redirect_url ) use ( &$redirected_url ) {
 		AMP_Validation_Error_Taxonomy::handle_single_url_page_bulk_and_inline_actions( $incorrect_post_type );
 		$this->assertEquals( get_term( $error_term->term_id )->term_group, $initial_accepted_status );
 
+		// Setup admin user which have edit_posts capability.
```

Comment:
The `edit_posts` capability or the `edit_terms` capability?

Reply:
I am not sure if there is any capability like `edit_terms`. It's rather an [arg](https://developer.wordpress.org/reference/functions/register_taxonomy/#:~:text=%27manage_categories%27.-,edit_terms,-string) in taxonomy capabilities.

Reply:
I guess `edit_term` is a meta capability and `manage_categories` is the primitive capability.